pkg/platform/openstack/loadbalancer.go

```diff
@@ -411,3 +411,34 @@ func listOpenStackOctaviaProviders(client *gophercloud.ServiceClient) ([]provide
 		return providersList, nil
 	}
 }
+
+// Iterate on pool members and check their address against provided list.
+// Remove all surplus members, which address doesn't existst on that list.
```

Comment:
typo: exist. Nit: better to specify what is the provided list.

Reply:
ack.